accounts/serializers.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import serializers
+from django.contrib.auth import authenticate
+from .models import Account
+from django.core.exceptions import ValidationError
+
+class AccountSerializer(serlializers.ModelSerializer):
+    """
+    This class does the job of a ModelForm as well as 
+    serializes the users. 
+    """
+
+    class Meta:
+        model = Account
+        fields = ["email", "password", "date_joined", "last_login", "hospital",]
+
+
+    def create(self, validated_data):
+        cleaned_email = validated_data["email"]
+        cleaned_password = validated_data["password"]
+
+        acct = Account
```

Comment:
What does this line do?

Reply:
Ahh, I think this must've snuck in from another development branch. This is an incomplete (still in progress) serializer for the Account model. It should be removed for this PR.